boot/bootstate20.go

```diff
@@ -292,6 +319,12 @@ func (ks20 *bootState20Kernel) markSuccessful(update bootStateUpdate) (bootState
 
 	// u should always be non-nil if err is nil
 	u.bootedKernelSnap = sn
+
+	// always set the commit status to be Default because we need to finish this
+	// "boot"
+	// TODO:UC20: should this really be hard-coded in commit instead?
```

Comment:
I still think that this is useful only if we tried to have one commit implementation (I think it's actually probably possible, and wouldn't be completely unreadable but not a goal as such).  Otherwise is just passing/using state which is a constant in fact. Also we aren't doing this for base, and I don't see a reason to.